src/main/java/org/triplea/ui/TripleaMainMenu.java

```diff
@@ -0,0 +1,174 @@
+package org.triplea.ui;
+
+import java.awt.Desktop;
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.text.MessageFormat;
+
+import games.strategy.debug.ClientLogger;
+import games.strategy.engine.ClientContext;
+import games.strategy.triplea.UrlConstants;
+import javafx.fxml.FXML;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.control.Button;
+import javafx.scene.control.PasswordField;
+import javafx.scene.control.TextField;
+import javafx.scene.input.MouseEvent;
+import javafx.scene.layout.BorderPane;
+import javafx.scene.layout.HBox;
+import javafx.scene.layout.VBox;
+import javafx.scene.text.Text;
+
+public class TripleaMainMenu extends BorderPane {
```

Comment:
A question about these subordinate visual classes...  Do they need to be public?  Are they only going to be composed by `TripleA` or other classes in this package?  Just wondering if we need to prefix every class with `Triplea`.